server/bleep/src/query/stopwords.rs

```diff
@@ -0,0 +1,48 @@
+use lazy_regex::regex;
+use once_cell::sync::OnceCell;
+use std::collections::HashSet;
+
+type StopWords = HashSet<String>;
+static STOPWORDS: OnceCell<StopWords> = OnceCell::new();
+static STOP_WORDS_LIST: &str = include_str!("stopwords.txt");
+
+fn stop_words() -> &'static StopWords {
+    STOPWORDS.get_or_init(|| {
+        let mut sw = StopWords::new();
+        for w in STOP_WORDS_LIST.lines() {
+            sw.insert(w.to_string());
+        }
+        sw
+    })
+}
+
+fn phrases<'a>(phrases_iter: impl IntoIterator<Item = &'a str>) -> Vec<Vec<&'a str>> {
+    let phrases_iter = phrases_iter.into_iter();
+    let mut phrases = Vec::with_capacity(2 * phrases_iter.size_hint().0);
+    for s in phrases_iter.filter(|s| !s.is_empty()) {
+        let mut phrase = Vec::new();
+        for word in s.split_whitespace() {
+            if stop_words().contains(&word.to_lowercase()) {
```

Comment:
maybe we should initialize stop_words once outside the for loops using a single call to stop_words()